recipes/genenotebook/build.sh

```diff
@@ -5,9 +5,30 @@ set -exo pipefail
 outdir=${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}-${PKG_BUILDNUM}
 mkdir -p $outdir $PREFIX/bin
 pwd
-ls -l .
+ls -lah .
+
+CC --version || echo 'cc not found'
+GCC --version || echo 'gcc not found'
+
+git clone https://github.com/meteor/meteor
+
+pushd meteor
+git checkout 39cc07472f8f1dc820b29b8c5a30813051d62700 #V1.9.0 commit
```

Comment:
Is this otherwise a submodule in github? Do you know if this is available in conda-forge? Maybe we could add it as a dependency if so (I unfortunately know nothing about meteor, so I don't know if this is feasible).

Reply:
Meteor is not in conda-forga afaik. Getting it to play nice with the build environment was a bit of a struggle, hence this approach. Typically meteor is installed by downloading a prebuilt package (https://www.meteor.com/developers/install). Not sure what you mean with it otherwise being a submodule, do you mean I should add it as a submodule to the genenotebook repository? I would prefer to have this merged as is, as I am staging a new release using meteor 2.0 that hopefully can use the regular build process.